src/app/shipping/Shipping.tsx

```diff
@@ -318,6 +318,8 @@ export function mapToShippingProps({
         countriesWithAutocomplete.push('GB');
     }
 
+    const shippingAddress = !shouldShowMultiShipping && consignments.length > 1 ? undefined : getShippingAddress();
```

Comment:
Just curious, what's the reasoning behind showing an empty address form? Is it to force the shopper to re-enter their shipping details? What would happen to the second consignment (and others) once they do so? Would they still stick around but not visible to the shopper?

Reply:
The empty address is to force the shopper to enter one. Since there are two consignments and both could have valid addresses, the shipping step cannot pick one, so instead it asks the shopper to re enter the information.

Reply:
Ok, thank you, that's good - they get combined and the shopper can resume their checkout in an expected state once they re-enter their address details 👍